pkg/client/library/push.go

```diff
@@ -95,7 +95,7 @@ func UploadImage(filePath string, libraryRef string, libraryURL string, authToke
 		}
 		sylog.Debugf("Upload completed OK\n")
 	} else {
-		sylog.Infof("Image is already present in the library - not uploading.\n")
+		return fmt.Errorf("image (%q) is already present in the library: not uploading", filePath)
```

Comment:
This isn't an error that should be called with `Errorf` should it? I mean, if you give it the same image, but a different tag, the image isn't uploaded, but the tag is created.

Reply:
Oh yeah, Let me see if I can fix this up... :+1:

Reply:
@jmstover, Better now?

Reply:
Hrmm... This is where other folks need to chime in (@jscook2345 ??). I think a `Infof is perfectly acceptable here, and ... it isn't really a warning. It's just an informational message about the upload...

Reply:
And that was modified in between me starting to write this and actually finishing it... :P